specshaper/encrypt-bundle/1.1/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+    "container": {
+        "encrypt_key": "%env(GDPR_ENCRYPT_KEY)%"
+    },
+    "env": {
+        "GDPR_ENCRYPT_KEY": "%generate(secret, 32)%"
```

Comment:
Shouln't you pass this to your bundles config instead of adding it to the container like that?